code/modules/roguetown/roguecrafting/ceramics.dm

```diff
@@ -0,0 +1,72 @@
+/datum/crafting_recipe/roguetown/ceramics
+	skillcraft = /datum/skill/misc/ceramics
+
+/datum/crafting_recipe/roguetown/ceramics/clay
+	structurecraft = /obj/structure/fluff/ceramicswheel
```

Comment:
Do these recipes show up in-game or is there something preventing this?

Reply:
Those recipes do not show up in game. They exist in the game code and can be ''''crafted'''' (If only by calling it manually through an in-game debug panel), but they **do not appear as a craftable option for the player under any circumstances**. I think it has something to do with lacking a result or lacking a name. In any case I think that this is handled by the datum/crafting_recipe class or its child datum/crafting_recipe/roguetown/.

These first three crafting recipes are there only to reduce the amount of repetition that will be needed for clay and glass items, since having a variable assignment to structurecraft and skillcraft for every clay item, when all of them will use the potter's wheel and ceramics skill, would make the code unnecessarily dense.

Reply:
That's good
> These first three crafting recipes are there only to reduce the amount of repetition that will be needed for clay and glass items, since having a variable assignment to structurecraft and skillcraft for every clay item, when all of them will use the potter's wheel and ceramics skill, would make the code unnecessarily dense.
Yeah, I get that, just your usual OOP